AppCenter/AppCenter/Internals/Util/MSEncrypter.m

```diff
@@ -76,61 +76,70 @@ - (void)dealloc {
 }
 
 - (NSString *_Nullable)encryptString:(NSString *)string {
+  NSData *dataToEncrypt = [string dataUsingEncoding:NSUTF8StringEncoding];
+  return [[self encryptData:dataToEncrypt] base64EncodedStringWithOptions:0];
```

Comment:
Please use a constant https://developer.apple.com/documentation/foundation/nsdatabase64encodingoptions?language=objc

Reply:
There are no options needed + I don't want to change the behavior here